src/api/schema.prisma

```diff
@@ -344,7 +344,9 @@ model Broker {
   id                      String        @id @default(cuid())
   application             Application?  @relation("Broker_application", fields: [applicationId], references: [id])
   applicationId           String?       @map("application")
+  buildingNumberOrName    String        @default("")
   isUsingBroker           Boolean?
+  isBasedInUk             Boolean?
```

Comment:
Should there be a default i.e. false?

Reply:
No, otherwise when a user loads the page, an answer of "no" would be selected by default. We need the user to provide a yes/no answer. 

We have this approach in many places - lots of existing yes/no questions.